src/schema/posts.ts

```diff
@@ -880,16 +894,12 @@ export const resolvers: IResolvers<any, Context> = {
     },
     submitExternalLink: async (
       _,
-      {
-        sourceId,
-        url,
-        commentary,
-      }: { sourceId: string; url: string; commentary: string },
+      { sourceId, commentary, url, title, image }: SubmitExternalLinkArgs,
       ctx,
     ): Promise<GQLEmptyResponse> => {
-      await ctx.con.transaction(async (manager) => {
-        await ensureSourcePermissions(ctx, sourceId, SourcePermissions.Post);
+      await ensureSourcePermissions(ctx, sourceId, SourcePermissions.Post);
```

Comment:
Why did you move this outside of the transaction?

Reply:
Oh, was working on something, and ended up removing it. This got left. Though this is not utilized/needed in the transaction so nothing should be different performance-wise. Probably even better since we don't have to initialize the transaction while it is not yet needed.

Reply:
Does query, so would prefer it to stay in the transaction to not loose scope.

Reply:
Oh, cool. Let's keep it inside.

Reply:
It's still outside 👀

Reply:
Too many PRs going left and right. Forgive me this 😭